java/arcs/core/storage/api/Entity.kt

```diff
@@ -24,4 +27,11 @@ interface Entity {
 interface EntitySpec<T : Entity> {
     /** Returns an empty new instance of [T]. */
     fun create(): T
+
+    /**
+     * Takes a [RawEntity] representing a serialized representation of a [RawEntity] from the
```

Comment:
I think there's still some redundant stuff here. Just: `Takes a [RawEntity] and convert it to concrete entity class [T].`